app/incident/views.py

```diff
@@ -0,0 +1,277 @@
+# Replace with your User model import
+from django.contrib.auth import (
+    get_user_model
+)
+from django.http import JsonResponse
+from django.views import View
+import json
+from core.utils import upload_file_to_s3
+from core.models import IncidentImage, Incident
+from core.models import IncidentCategory
+from django.shortcuts import get_object_or_404
+from http import HTTPStatus
+from .messages import MESSAGES
+
+
+# API Logic for Incident Category
+class IncidentCategoryView(View):
+    def get(self, request):
+        lng = 'en-CA'
+        try:
+            categories = IncidentCategory.objects.all()
+            category_list = []
+            for category in categories:
+                category_list.append({
+                    'id': category._id,
+                    'name': category.name,
+                })
+            return JsonResponse({
+                'message': MESSAGES[lng].
+                get('SUCCESS_MESSAGE_FOR_INCIDENT_CATEGORY'),
+                'data': category_list,
+                'error': False,
+                'status': HTTPStatus.OK
+            }, status=HTTPStatus.OK)
+        except Exception as e:
+            return JsonResponse({
+                'message': MESSAGES[lng]
+                .get('ERROR_MESSAGE_INVALID_TOKEN').format(str(e)),
```

Comment:
Don't pass the actual stack trace of exception, Your error message should have been "Some error occurred, please try again later"

Reply:
Everywhere you did the same thing, Giving stack trace to end user is like shooting your app vulnerabilities to end user.